sdk/iot/Azure.Iot.Hub.Service/src/Generated/Models/DevicesStatistics.cs

```diff
@@ -8,18 +8,18 @@
 namespace Azure.Iot.Hub.Service.Models
 {
     /// <summary> The RegistryStatistics. </summary>
-    public partial class RegistryStatistics
+    public partial class DevicesStatistics
     {
-        /// <summary> Initializes a new instance of RegistryStatistics. </summary>
-        internal RegistryStatistics()
+        /// <summary> Initializes a new instance of DevicesStatistics. </summary>
+        internal DevicesStatistics()
         {
         }
 
-        /// <summary> Initializes a new instance of RegistryStatistics. </summary>
+        /// <summary> Initializes a new instance of DevicesStatistics. </summary>
         /// <param name="totalDeviceCount"> The total number of devices registered for the IoT Hub. </param>
         /// <param name="enabledDeviceCount"> The number of currently enabled devices. </param>
         /// <param name="disabledDeviceCount"> The number of currently disabled devices. </param>
-        internal RegistryStatistics(long? totalDeviceCount, long? enabledDeviceCount, long? disabledDeviceCount)
+        internal DevicesStatistics(long? totalDeviceCount, long? enabledDeviceCount, long? disabledDeviceCount)
```

Comment:
Are these fields ever actually null? Seems like we could convince the service folks to change the swagger to make these not nullable

Reply:
I haven't been able to produce null results so far, but during the design discussion we came to the conclusion that there might be circumstances in which a null value can happen. I will double check with the service folks.